### ISSUE TITLE
Latest release is broken on Ubuntu 16.04

### ISSUE THREAD

**Issue by henroth** [catkin/catkin_tools#613]
Hi,

First off thanks for catkin_tools, as a long time ROS user it is a huge improvement from catkin_make.

The latest release (0.6.0) seems to have broken the packages for Ubuntu16.04. I am using docker, but I believe this is relevant for a normal install. My docker file is this 

```
FROM ros:kinetic-perception
RUN apt-get update && apt-get upgrade -y && apt-get install -y python-catkin-tools
```

### System Info

```
root@593cf0e3210b:/# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
root@593cf0e3210b:/# python --version
Python 2.7.12
root@593cf0e3210b:/# echo $ROS_DISTRO 
kinetic
```

### Build / Run Issue

Here is the output I get from running 
apt-get update && apt-get upgrade -y && apt-get install -y python-catkin-tools

```bash
hroth-ub:catkin_tools$ docker build .
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM ros:kinetic-perception
 ---> 41c6b3b2857b
Step 2/2 : RUN apt-get update && apt-get upgrade -y && apt-get install -y python-catkin-tools
 ---> Running in be227f00e0e5
Get:1 http://packages.ros.org/ros/ubuntu xenial InRelease [4678 B]
Get:2 http://archive.ubuntu.com/ubuntu xenial InRelease [247 kB]
Get:3 http://security.ubuntu.com/ubuntu xenial-security InRelease [109 kB]
Get:4 http://packages.ros.org/ros/ubuntu xenial/main amd64 Packages [837 kB]
Get:5 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [109 kB]
Get:6 http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages [1117 kB]
Get:7 http://archive.ubuntu.com/ubuntu xenial-backports InRelease [107 kB]
Get:8 http://archive.ubuntu.com/ubuntu xenial/main amd64 Packages [1558 kB]
Get:9 http://archive.ubuntu.com/ubuntu xenial/restricted amd64 Packages [14.1 kB]
Get:10 http://archive.ubuntu.com/ubuntu xenial/universe amd64 Packages [9827 kB]
Get:11 http://security.ubuntu.com/ubuntu xenial-security/restricted amd64 Packages [12.7 kB]
Get:12 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 Packages [628 kB]
Get:13 http://security.ubuntu.com/ubuntu xenial-security/multiverse amd64 Packages [6677 B]
Get:14 http://archive.ubuntu.com/ubuntu xenial/multiverse amd64 Packages [176 kB]
Get:15 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 Packages [1483 kB]
Get:16 http://archive.ubuntu.com/ubuntu xenial-updates/restricted amd64 Packages [13.1 kB]
Get:17 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages [1031 kB]
Get:18 http://archive.ubuntu.com/ubuntu xenial-updates/multiverse amd64 Packages [19.7 kB]
Get:19 http://archive.ubuntu.com/ubuntu xenial-backports/main amd64 Packages [7942 B]
Get:20 http://archive.ubuntu.com/ubuntu xenial-backports/universe amd64 Packages [8807 B]
Fetched 17.3 MB in 3s (4495 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Calculating upgrade...
The following packages will be upgraded:
  apt ca-certificates libapt-pkg5.0 libssl-dev libssl1.0.0 libsystemd0
  libudev1 openssl systemd systemd-sysv
10 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 8778 kB of archives.
After this operation, 35.8 kB disk space will be freed.
Get:1 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libsystemd0 amd64 229-4ubuntu21.28 [206 kB]
Get:2 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 systemd amd64 229-4ubuntu21.28 [3639 kB]
Get:3 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libapt-pkg5.0 amd64 1.2.32ubuntu0.1 [713 kB]
Get:4 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 apt amd64 1.2.32ubuntu0.1 [1087 kB]
Get:5 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 systemd-sysv amd64 229-4ubuntu21.28 [12.3 kB]
Get:6 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libudev1 amd64 229-4ubuntu21.28 [54.8 kB]
Get:7 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libssl-dev amd64 1.0.2g-1ubuntu4.16 [1344 kB]
Get:8 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libssl1.0.0 amd64 1.0.2g-1ubuntu4.16 [1084 kB]
Get:9 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 openssl amd64 1.0.2g-1ubuntu4.16 [492 kB]
Get:10 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 ca-certificates all 20190110~16.04.1 [146 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 8778 kB in 2s (3710 kB/s)
(Reading database ... 64497 files and directories currently installed.)
Preparing to unpack .../libsystemd0_229-4ubuntu21.28_amd64.deb ...
Unpacking libsystemd0:amd64 (229-4ubuntu21.28) over (229-4ubuntu21.27) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
Setting up libsystemd0:amd64 (229-4ubuntu21.28) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
(Reading database ... 64497 files and directories currently installed.)
Preparing to unpack .../systemd_229-4ubuntu21.28_amd64.deb ...
Unpacking systemd (229-4ubuntu21.28) over (229-4ubuntu21.27) ...
Setting up systemd (229-4ubuntu21.28) ...
Initializing machine ID from random generator.
addgroup: The group `systemd-journal' already exists as a system group. Exiting.
Operation failed: No such file or directory
(Reading database ... 64497 files and directories currently installed.)
Preparing to unpack .../libapt-pkg5.0_1.2.32ubuntu0.1_amd64.deb ...
Unpacking libapt-pkg5.0:amd64 (1.2.32ubuntu0.1) over (1.2.32) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
Setting up libapt-pkg5.0:amd64 (1.2.32ubuntu0.1) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
(Reading database ... 64497 files and directories currently installed.)
Preparing to unpack .../apt_1.2.32ubuntu0.1_amd64.deb ...
Unpacking apt (1.2.32ubuntu0.1) over (1.2.32) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
Setting up apt (1.2.32ubuntu0.1) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
(Reading database ... 64497 files and directories currently installed.)
Preparing to unpack .../systemd-sysv_229-4ubuntu21.28_amd64.deb ...
Unpacking systemd-sysv (229-4ubuntu21.28) over (229-4ubuntu21.27) ...
Setting up systemd-sysv (229-4ubuntu21.28) ...
(Reading database ... 64497 files and directories currently installed.)
Preparing to unpack .../libudev1_229-4ubuntu21.28_amd64.deb ...
Unpacking libudev1:amd64 (229-4ubuntu21.28) over (229-4ubuntu21.27) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
Setting up libudev1:amd64 (229-4ubuntu21.28) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
(Reading database ... 64497 files and directories currently installed.)
Preparing to unpack .../libssl-dev_1.0.2g-1ubuntu4.16_amd64.deb ...
Unpacking libssl-dev:amd64 (1.0.2g-1ubuntu4.16) over (1.0.2g-1ubuntu4.15) ...
Preparing to unpack .../libssl1.0.0_1.0.2g-1ubuntu4.16_amd64.deb ...
Unpacking libssl1.0.0:amd64 (1.0.2g-1ubuntu4.16) over (1.0.2g-1ubuntu4.15) ...
Preparing to unpack .../openssl_1.0.2g-1ubuntu4.16_amd64.deb ...
Unpacking openssl (1.0.2g-1ubuntu4.16) over (1.0.2g-1ubuntu4.15) ...
Preparing to unpack .../ca-certificates_20190110~16.04.1_all.deb ...
Unpacking ca-certificates (20190110~16.04.1) over (20170717~16.04.2) ...
Processing triggers for libc-bin (2.23-0ubuntu11) ...
Setting up libssl1.0.0:amd64 (1.0.2g-1ubuntu4.16) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Setting up libssl-dev:amd64 (1.0.2g-1ubuntu4.16) ...
Setting up openssl (1.0.2g-1ubuntu4.16) ...
Setting up ca-certificates (20190110~16.04.1) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Processing triggers for libc-bin (2.23-0ubuntu11) ...
Processing triggers for ca-certificates (20190110~16.04.1) ...
Updating certificates in /etc/ssl/certs...
10 added, 31 removed; done.
Running hooks in /etc/ca-certificates/update.d...
done.
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python-catkin-tools : Depends: python2 (< 2.8) but it is not installable
                       Depends: python2 (>= 2.7) but it is not installable
                       Depends: python2:any (>= 2.6.6-7~) but it is not installable
E: Unable to correct problems, you have held broken packages.
The command '/bin/sh -c apt-get update && apt-get upgrade -y && apt-get install -y python-catkin-tools' returned a non-zero code: 100
```

### Expected Behavior
The package to be installed

### Actual Behavior
The package is not installed

### Steps to Reproduce the Issue

```bash
sudo apt-get update && sudo apt-get upgrade -y && sudo apt-get install -y python-catkin-tools
```

**Comment by InspireX96** [comment]
I have the same issue running in an Ubuntu16.04 Docker

```
root@50b426b422f0:~# sudo apt install python-catkin-tools
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python-catkin-tools : Depends: python2 (< 2.8) but it is not installable
                       Depends: python2 (>= 2.7) but it is not installable
                       Depends: python2:any (>= 2.6.6-7~) but it is not installable
E: Unable to correct problems, you have held broken packages.
root@50b426b422f0:~# 
root@50b426b422f0:~# python -V
Python 2.7.12
root@50b426b422f0:~# python2 -V
Python 2.7.12
```

Also tried in an Ubuntu16.04 Virtualbox VM. Also got the dependency problem

**Comment by andre-nguyen** [comment]
Can you guys try again? It got magically solved on my end.

**Comment by dirk-thomas** [comment]
This has been resolved already with the release of version 0.6.1. See https://github.com/catkin/catkin_tools/issues/594#issuecomment-638479931
